packages/core/src/util/gestureUtils.ts

```
@@ -86,7 +86,7 @@ import type Cell from '../view/cell/Cell';
  */
 export const makeDraggable = (
   element: Element,
-  graphF: Graph | Function,
+  graphF: AbstractGraph | Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)